src/styles/app.css

```diff
@@ -123,6 +123,22 @@ input.tm-field {
   padding: 0.75rem;
 }
 
+#notifications-button {
```

Comment:
move to app.vue

Reply:
app.vue just imports this file

Reply:
But conceptionally app.css should only hold global styles. Any styles that target specific elements should go where the elements live. This reduces the mental overhead of needing to search for where the styles are defined.